media/doc/Romanian translation notes.txt

```diff
@@ -1,53 +1,15 @@
 The following material is addressed (in Romanian) to Romanian translators.
 
 Copyright 2014 Ștefan Fulea
-Copyright 2023 Miloiu Andrei
+Copyright 2023-2024 Miloiu Andrei
 
 ------------------------------------------------------------------------------
 
-* Notă introductivă
+ 1. We use only the correct Romanian language diacritics.
```

Comment:
```suggestion
 1. We use only the correct Romanian language diacritics.
     This means using lowercase ţ instead of ț.
     This means using lowercase ş instead of ș. The upper-case character to use for that is Ş.
```